lib/new_google_recaptcha.rb

```diff
@@ -8,8 +8,8 @@ def self.setup
     yield(self)
   end
 
-  def self.human?(token, model = nil)
-    is_valid = NewGoogleRecaptcha::Validator.valid?(token)
+  def self.human?(token, minimum_score, action, model = nil)
```

Comment:
a few questions: 
1) what about having a default value for minimum_score and having it configurable in an initializer?
2) why we need "action"? this is the same action as we specified in view?

Reply:
3. we need to have all this documented in readme

Reply:
For 2 just update readme, so it will be clear how it works.
Will wait for an updated PR and than merge